code/datums/elements/waddling.dm

```diff
@@ -1,9 +1,13 @@
 /datum/element/waddling
+	element_flags = ELEMENT_BESPOKE
+	argument_hash_start_idx = 2
+	var/hops = FALSE
 
-/datum/element/waddling/Attach(datum/target)
+/datum/element/waddling/Attach(datum/target, hops = FALSE)
```

Comment:
hops should be a more generic `animation_style` and pass in a define for various waddle styles